collect_app/src/androidTest/java/org/odk/collect/android/regression/FillBlankFormTest.java

```diff
@@ -158,7 +158,7 @@ public void sortByDialog_ShouldBeTranslatedAndDisplayProperIcons() {
 
         //TestCase37
         MainMenu.clickOnMenu();
-        MainMenu.clickGeneralSettings();
+        MainMenu.clickGeneralSettings(main.getActivity());
```

Comment:
Isn't it a context of MainMenuActivity? Probably it would work as well in this case but seems strange. What about using [ActivityHelpers.getActivity()](https://github.com/opendatakit/collect/blob/master/collect_app/src/androidTest/java/org/odk/collect/android/support/ActivityHelpers.java#L22) which returns the current activity?

Reply:
Oh we could use that in future though. I knew there must be a way of getting the "current" Activity rather than just the one on the test rule. That'll be useful thanks!